guide/01-getting-started/install-and-set-up.ipynb

```diff
@@ -395,7 +395,7 @@
     "\n",
```

Comment:
<blockquote>a total collection of 99 package</blockquote><p>should we keep the number of package here ?</p><p>The deep learning frameworks page already has this info.</p><p>which is maintained with high frequency.</p><p> </p><p>And with each release this number, keeps on changing.</p><p>Then we have to keep-up with updating this, at the same frequency.</p><p>I recommend removing the info about number of packages, as it is already present on the webpage.</p><p> </p><blockquote>Deep learning installer is now</blockquote><p>This should be consistent with the marketing name/term of the installer.</p><p>so we can go with <em>deep learning framework</em> or <em>Deep Learning Libraries Installer</em>.</p><p> </p><p> </p><p> </p><blockquote>adds packages to the default <code>arcgispro-py3</code>&nbsp;environment.</blockquote><p>We can remove name of default env. as the MSI installer will install it in active env. that can be changed using pro's python manager or proswap.</p><p>so it will be always be arcgispro-py3 unless it is selected as active env.</p><p> </p><p>This is also mentioned in the webpage as this:</p><p> </p><p><em>"Note that this will install the deep learning frameworks into your currently active Pro Python environment; if you wish to install in an environment aside from the default&nbsp;</em><code><em>arcgispro-py3</em></code><em>&nbsp;environment, switch to that environment before running the MSI using either ArcGIS Pro's Python manager UI or on the command line with&nbsp;</em><code><em>proswap</em></code><em>.&nbsp;"</em></p><p> </p><p>so we can follow a similar wording for this, as quoted above.</p><p> </p><p> </p><blockquote>&nbsp;install deep learning installer</blockquote><p>we should use the marketed term.</p><p> </p><p>Whatever marketing term is used, that should be consistent for the whole document, if it is d<em>eep learning framework</em> or <em>Deep learning libraries installer</em>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1067/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>